### ISSUE TITLE
I want to do something for the Project.Among them, visual editing behavior, Chinese documents

### ISSUE THREAD

**Issue by wqaetly** [meniku/NPBehave#21]
Hello, I'm shocked by your NPBehave.So I want to do something for him.Among them, visual editing behavior, Chinese documents (because I am a developer from China), please stay tuned!
I will finish it soon。

**Comment by meniku** [comment]
thanks, really looking forward to see what you come up with. 
Use pull requests when you want to contribute to the project and I'll review and integrate them.

Let me know if you need further help.

**Comment by wqaetly** [comment]
> thanks, really looking forward to see what you come up with.
> Use pull requests when you want to contribute to the project and I'll review and integrate them.
> 
> Let me know if you need further help.

Ok, now I have finished the Chinese document, I will upload it to you soon

**Comment by wqaetly** [comment]
Could you please leave your contact information? I feel that email is not a convenient way to communicate.




------------------ 原始邮件 ------------------
发件人: "NiKu"<notifications@github.com>;
发送时间: 2019年8月19日(星期一) 晚上8:03
收件人: "meniku/NPBehave"<NPBehave@noreply.github.com>;
抄送: "烟雨迷离半世殇"<1778139321@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [meniku/NPBehave] I want to do something for the Project.Amongthem, visual editing behavior, Chinese documents (#21)




thanks, really looking forward to see what you come up with.
 Use pull requests when you want to contribute to the project and I'll review and integrate them.
 
Let me know if you need further help.

**Comment by meniku** [comment]
thanks for your first contribution (Chinese README). I've merged it. The only thing I changed was the URLs for the logo and debugger doc.

Regarding the contact-information, just send me an E-Mail and I'll hand it over to you.

**Comment by wqaetly** [comment]
Ok, now that I've made the first version of NPBehave editor, I can do everything I need to do to export binary data for the server to read, and the rest is to optimize the node GC (remove all constructors, pool node classes)，coming soon！
![QQ截图20190909222237](https://user-images.githubusercontent.com/35335061/64539203-a58f7f00-d350-11e9-904f-d2fa21aea24b.png)

**Comment by wqaetly** [comment]
This is my copy of your blackboard node using the example, it works
![-380fd3061ef3d289](https://user-images.githubusercontent.com/35335061/64539950-f81d6b00-d351-11e9-87ec-5594c4dafa19.png)
![QQ截图20190909222237](https://user-images.githubusercontent.com/35335061/64539952-f8b60180-d351-11e9-9653-a52d6234289b.png)

**Comment by meniku** [comment]
pretty amazing! You could consider to attach decorator nodes directly at the decorated nodes ( as in the behaviour trees editor of UE4 ), else the graph might grow up pretty quick.

Great Job!

**Comment by wqaetly** [comment]
Sorry, I don't quite understand what you mean by decorator connecting decorator directly. This is the screenshot of unreal 4, is that what you mean
![PESFG {VU SI%D18$FUCNC8](https://user-images.githubusercontent.com/35335061/64661049-a968f180-d475-11e9-91a4-8aaa0c1240d1.png)

**Comment by meniku** [comment]
yep exactly. See how they put the Decorators (Blue) right at the decorated composites ( gray )? 

It's just an suggestion.

The NPBehave Debugger does it also
![Debugger](https://github.com/meniku/NPBehave/blob/master/README-Debugger.png?raw=true)

**Comment by wqaetly** [comment]
Well, in unreal 4 it's decoration + composition, but I don't feel the need and I feel the difficulty, because it would be intrusive to my current code structure




------------------ 原始邮件 ------------------
发件人: "NiKu"<notifications@github.com>;
发送时间: 2019年9月11日(星期三) 晚上7:05
收件人: "meniku/NPBehave"<NPBehave@noreply.github.com>;
抄送: "烟雨迷离半世殇"<1778139321@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [meniku/NPBehave] I want to do something for the Project.Amongthem, visual editing behavior, Chinese documents (#21)




yep exactly. See how they put the Decorators (Blue) right at the decorated composites ( gray )?
 
It's just an suggestion.
 
The NPBehave Debugger does it also

**Comment by meniku** [comment]
alright. Also we can still iterate once we have a first working version :)

**Comment by wqaetly** [comment]
OK,I have submitted the visual editor branch, please check

**Comment by wqaetly** [comment]
ok，Now Unity's GraphView is very popular，and i have use it to make npbehave visual

![image](https://user-images.githubusercontent.com/35335061/204204331-9e4c15d6-150f-4734-b52e-2c0450e98f9d.png)

Visualization is really important, it is easy to edit and debug. I will provide a visual editor based on native GraphView for NPBehave when I am free, and provide editing + debugging functions
